src/main/inAppNotifications/views/user/InAppUserSpaceCommunityInvitationView.tsx

```
@@ -5,22 +5,16 @@ export const InAppUserSpaceCommunityInvitationView = (notification: InAppNotific
   const { payload, triggeredBy } = notification;
 
   // do not display notification if these are missing
-  if (!triggeredBy?.profile?.displayName || !payload.space) {
+  if (!triggeredBy?.profile?.displayName || !payload.space?.about?.profile) {
     return null;
   }
-  const space = payload.space;
+  const { profile } = payload.space?.about;
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)